.github/workflows/docker-build-release.yml

```diff
@@ -1,9 +1,8 @@
-name: Publish Docker image
+name: Publish Docker images
 
 on:
-  push:
-    tags:
-      - "arize-phoenix-v*"
+  release:
+    types: [published]
```

Comment:
is this gonna trigger on evals though?

Reply:
that's a good point, it will, let me find a different strategy